src/coprocessor/mod.rs

```diff
@@ -16,10 +16,13 @@ mod metrics;
 mod dag;
 mod statistics;
 mod checksum;
+mod readpool_context;
```

Comment:
I prefer to use `context` directly

Reply:
we have too many contexts... :)  let's use a more accurate name to avoid confusion..